src/TimePicker/_TimePickerItem.js

```diff
@@ -201,10 +201,38 @@ class TimePickerItem extends Component {
         }
     };
 
-    onBlur = () => {
-        const { isValid } = this.state;
+    isValidTime = (value) => {
```

Comment:
What is different here than `onInputValidation`? Would it make sense it reuse that function/improve it instead of reinventing it?

Reply:
Thanks for your review comment.
regex for few below cases in onInputValidation were not matching input correctly.
eg.
a. 12-Hour Clock - (hh:mm:ss am/pm)
-> Here user can enter hour >= 13 and minute/second >= 60
b. 24-Hour Clock - (hh:mm:ss)
-> Here user can enter hour >= 24 and minute/second >= 60
c. Time Picker With No Seconds - (hh:mm)
-> Here user can enter minute/second >= 60
As you suggested I have modified the code to use onInputValidation function for change as well as blur event type.
Please review.